src/main/java/dev/ai4j/openai4j/chat/AssistantMessage.java

```diff
@@ -1,19 +1,32 @@
 package dev.ai4j.openai4j.chat;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSetter;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
 import java.util.List;
 import java.util.Objects;
 
 import static dev.ai4j.openai4j.chat.Role.ASSISTANT;
 import static java.util.Arrays.asList;
 import static java.util.Collections.unmodifiableList;
 
+@JsonDeserialize(builder = AssistantMessage.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
We also need to add `@JsonIgnoreProperties(ignoreUnknown = true)` everywhere to make sure deserialization will not fail when OpenAI will add some new fields.

Reply:
Added the annotation to accept unknown fields when deserializing JSON into the Java model.